server/src/routes/auth.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const router = express.Router();
+
+const User = require('../models/user.model');
+
+
+// PassportJS imports for auth
+const passport = require('passport');
+// Encryption library
+const crypto = require('crypto');
+
+
+// User logs in with password
+/**
+ * Note: you MUST use `username` as the JSON body key or else this will not work! 
+ * 
+ * Expected body parameters:
+ * @type string {body.username}
+ * @type string {body.password}
+ */
+router.post('/login/password', 

```

Comment:
/login alone should work instaed of /login/password

Reply:
Done!